Cargo.Bazel.Fuzzing.toml.lock

```diff
@@ -1802,12 +1804,13 @@ checksum = "37b2a672a2cb129a2e41c10b1224bb368f9f37a2b16b612598138befd7b37eb5"
 
 [[package]]
 name = "cc"
-version = "1.0.83"
+version = "1.2.1"
```

Comment:
assuming you checked that this includes the fix for https://github.com/rust-lang/cc-rs/issues/1271

Reply:
it is in